lib/notifier.rb

```diff
@@ -0,0 +1,6 @@
+require 'twilio-ruby'
```

Comment:
Next steps would be to connect this to the Twilio API: https://www.twilio.com/docs/usage/api